src/plugins/condition/components/Condition.vue

```diff
@@ -211,6 +212,50 @@ export default {
             criterionIndex: 0
         };
     },
+    computed: {
+        getSummary: function () {
+            let config = this.domainObject.configuration;
+
+            if (!config.criteria.length) {
+                return 'When all else fails';
+            } else {
+                let rule = '';
+                let summary = 'No criteria specified';
+                if (config.criteria.length === 1 && config.criteria[0].telemetry) {
+                    if (config.criteria[0].operation) {
+                        if (config.criteria[0].input.length ||
+                            (config.criteria[0].operation === 'isDefined' ||
+                             config.criteria[0].operation === 'isUndefined')) {
```

Comment:
this check is similar to one below and looks like it can be extracted and reused

Reply:
Redone with use of vue js conditional directives in markup